LunarTimes/TabBarViewController.swift

```diff
@@ -42,16 +44,23 @@ extension UIViewController {
 
 protocol LocationChangedDelegate {
     func locationUpdated(longitude: Double, latitude: Double, placemark: CLPlacemark?)
+    func locationUpdated(longitude: Double, latitude: Double)
```

Comment:
I dont think this is needed, you can pass in nil in placemark for the same effect.